libs/api/metadata-converter/src/lib/iso19139/write-parts.ts

```diff
@@ -255,6 +255,8 @@ export function getServiceDistributionProtocol(
       return 'OGC:WMS'
     case 'wps':
       return 'OGC:WPS'
+    case 'stac':
+      return 'STAC Items'
```

Comment:
See my other comments
```suggestion
      return 'STAC'
```

Reply:
Out of scope, I'm gonna remove this change.